controllers/topics.controller.js

```diff
@@ -0,0 +1,13 @@
+const { fetchTopic } = require("../models/topics.model");
+
+exports.getTopics = (req, res, next) => {
+    const { topics } = req.params;
+    fetchTopic(topics)
+    .then((topic) => {
+        console.log(topic);
+        res.status(200).send({ topics: topic })
+    })
+    .catch((err) => {
```

Comment:
Good habit to pass errors to `next` - I'd get an error handler in place though for it